hugegraph-test/src/main/java/com/baidu/hugegraph/unit/UnitTestSuite.java

```diff
@@ -19,18 +19,19 @@
 
 package com.baidu.hugegraph.unit;
 
-import org.junit.runner.RunWith;
-import org.junit.runners.Suite;
-
-import com.baidu.hugegraph.unit.cache.CacheManagerTest;
-import com.baidu.hugegraph.unit.cache.RamCacheTest;
 import com.baidu.hugegraph.unit.core.AnalyzerTest;
 import com.baidu.hugegraph.unit.core.BackendMutationTest;
 import com.baidu.hugegraph.unit.core.CassandraTest;
 import com.baidu.hugegraph.unit.core.ConditionQueryFlattenTest;
 import com.baidu.hugegraph.unit.core.EdgeIdTest;
 import com.baidu.hugegraph.unit.core.JsonUtilTest;
 import com.baidu.hugegraph.unit.core.VersionTest;
+import com.baidu.hugegraph.unit.core.StringUtilTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+import com.baidu.hugegraph.unit.cache.CacheManagerTest;
+import com.baidu.hugegraph.unit.cache.RamCacheTest;
```

Comment:
keep import in alphabetical order

Reply:
Imports are now in alphabetical order

Reply:
> 
> 
> Imports are now in alphabetical order

Maybe u can refer [huge-code-style-props](https://github.com/imbajin/nocode/blob/master/properties/hugegraph-style.xml) or import it if u use `IDEA`, can avoid most of package import, order and alignments issues..